sdk/communication/azure-communication-administration/tests/test_phone_number_administration_client.py

```diff
@@ -125,6 +128,19 @@ def setUp(self):
             self.capabilities_id = "capabilities_id"
             self.release_id = "release_id"
 
+    @pytest.mark.live_test_only
+    @pytest.mark.skipif(SKIP_PHONE_NUMBER_TESTS, reason=PHONE_NUMBER_TEST_SKIP_REASON)
+    def test_list_all_phone_numbers_from_managed_identity(self):
+        endpoint, access_key = parse_connection_str(self.connection_str)
+        from devtools_testutils import is_live
+        if not is_live():
+            credential = FakeTokenCredential()
+        else:
+            credential = DefaultAzureCredential()
+        phone_number_client = PhoneNumberAdministrationClient(endpoint, credential)
```

Comment:
Can you move this logic of creating a FakeToken Credential VS DefaultAzureCredential into the test utils too?
I see we are doing the same in this test and the async version of the test